src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.Intrinsics.cs

```diff
@@ -112,7 +112,7 @@ static IntrinsicHashtable InitializeIntrinsicHashtable()
             table.Add(CorInfoIntrinsics.CORINFO_INTRINSIC_GetRawHandle, "AllocatorOf", "System", "Activator");
 
             // If this assert fails, make sure to add the new intrinsics to the table above and update the expected count below.
-            Debug.Assert((int)CorInfoIntrinsics.CORINFO_INTRINSIC_Count == 34, "Please update intrinsic hash table");
+            Debug.Assert((int)CorInfoIntrinsics.CORINFO_INTRINSIC_Count == 32, "Please update intrinsic hash table");
```

Comment:
I think it would better to just delete this assert. It made sense when this was a separate project in CoreRT. It is not the case anymore.

Reply:
Good point, removed in 2nd commit.